Changelog.md

```diff
@@ -1,34 +1,42 @@
-## 4.0.0 - UNRELEASED
+## 4.0.0 - 2024-03-13
 
-## Changed
+### Added
+
+-   Reintroduced triggers functionality.
+-   The configuration in the advanced settings menu to stop saving to hard disk
+    when the HDD space is running low.
+-   The default path for the session can be changed from the advanced settings
+    menu.
+
+### Changed
 
 -   Changed export format to `.ppk2`. `.ppk` has been deprecated and importing
     this format will autogenerate a `.ppk2` file next to the `.ppk` file.
--   Exporting of data now works also when the data was loaded from file. This
-    allows partial export to `.csv`.
 -   Removed limitations for recording duration by using the disk to store the
     collected data instead of using RAM.
+-   Performance improvements.
 
-## Fixed
+### Fixed
 
--   Chart drifting when live mode and PC resources where strained.
+-   Chart drifting in live mode.
```

Comment:
My concern here is that we sole one instance of this from happening ... I know there are other cases that we were not able to solve unless we do a FW update

Reply:
Fixed:
* Some cases when chart was drifting in the live mode.
--
Maybe like this?